templates/lxc-ubuntu.in

```diff
@@ -92,7 +92,15 @@ configure_ubuntu()
     password=$5
 
     # configure the network using the dhcp
-    cat <<EOF > $rootfs/etc/network/interfaces
+    if chroot $rootfs which netplan; then
```

Comment:
You should redirect the output of which to /dev/null. We usually use "which blah >/dev/null 2>&1" in our scripts.